main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+def main
+  app = App.new
+  puts "Welcome to School Library App!\n\n"
+  app.run
+end
+
```

Comment:
- [ ] I am not able to run the application please address that:
![main](https://user-images.githubusercontent.com/44745855/180621400-73d0284f-37d9-437c-8484-41a93e22d3d0.png)